calibration/CTDMOG/CGINS-CTDMOG-13601__20170401.csv

```diff
@@ -11,15 +11,15 @@ serial,name,value,notes
 37-13601,CC_j,3.026342e-005,
 37-13601,CC_p_range,1450,
 37-13601,CC_pa0,4.009798e-001,
-37-13601,CC_pa1,4.657163e-003,
+37-13601,CC_pa1,4.657164e-03,
 37-13601,CC_pa2,-3.332243e-011,
 37-13601,CC_ptca0,5.248207e+005,
-37-13601,CC_ptca1,-6.974895e-001,
+37-13601,CC_ptca1,-6.974896e-01,
 37-13601,CC_ptca2,2.039662e-001,
 37-13601,CC_ptcb0,2.497200e+001,
 37-13601,CC_ptcb1,0.00E+00,
 37-13601,CC_ptcb2,0.00E+00,
 37-13601,CC_ptempa0,1.661177e+002,
 37-13601,CC_ptempa1,-6.094802e-002,
 37-13601,CC_ptempa2,-1.956393e-006,
-37-13601,CC_wbotc,7.7547e-007,
+37-13601,CC_wbotc,7.754676e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT